tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
This is a good way to structure the tests but I would usually call something like this `userdatadict` a "Fixture". So can we just rename this module to `fixutres/userdata.py`? That will be a lot clearer and more specific than `conftest.py`.

Reply:
yeahh.... my bad.. I go 10000km/hr at all times: adhd needing intensity and all: I meant to circle back on that but it got lost in the excitement